src/patternfly/components/MenuToggle/menu-toggle-button.hbs

```diff
@@ -0,0 +1,16 @@
+{{#if menu-toggle--IsExpanded}}
```

Comment:
Can you wrap the contents of this partial in a menu toggle BEM element? It's generally bad practice for a component (BEM block) to also be a BEM element somewhere else. 


```
<div class="pf-c-menu-toggle__button{{#if menu-toggle-button--modifier}} {{menu-toggle-button--modifier}}{{/if}}"
  {{#if menu-toggle-button--attribute}}
    {{{menu-toggle-button--attribute}}}
  {{/if}}>
  // contents of this partial go here
</div>
```